test/utils/locators.ts

```diff
@@ -1,3 +1,9 @@
 export function toolpadHomeAppRow(appName: string): string {
```

Comment:
We should get rid of this file and move the locators inside of the `ToolpadHome` and `ToolpadEditor` models

Reply:
yeah im gonna use those new objects, just havent done it yet